libdrgn/program.c

```diff
@@ -99,7 +99,10 @@ void drgn_program_deinit(struct drgn_program *prog)
 		else
 			drgn_thread_set_deinit(&prog->thread_set);
 	}
-	drgn_thread_destroy(prog->crashed_thread);
+	// When the target is a userspace core dump, `crashed_thread` will have been
+	// included in `prog->thread_set`, and thus freed by the above call to `drgn_thread_set_deinit`
```

Comment:
I was going to nitpick the `//`-style comments and tell you to use `/* */`, but honestly that's a rule I inherited from the Linux kernel style, which only has that rule because it targets C89, and C89 doesn't have `//`-style comments (although GCC seems to allow it in GNU89 mode now). So I think that's a pointless rule these days, and this is me giving up on it. You can leave it as `//`.

However, I would still like this comment to be wrapped to 80 characters.